src/_tutorials/web/low-level-html/connect-dart-html.md

```diff
@@ -394,26 +422,26 @@ Most HTML uses cascading style sheets (CSS) to define _styles_
 that control the appearance of page elements.
 Let's customize the CSS for the mini app.
 
-1. Click **CSS**.
+1. Click **CSS**, at the upper left of DartPad.
    The view switches from Dart code to the (non-existent) CSS code.
 
 2. Add the following CSS code:
 
-   {% prettify none tag=pre+code %}
+   {% prettify css tag=pre+code %}
```

Comment:
Looks much better with the formatting!

But the only time I can remember choosing to use prettify is when I use highlighting. (<code>```css</code> doesn't work?)